.github/workflows/ci.yml

```diff
@@ -33,7 +33,9 @@ jobs:
       - name: Setup .NET SDK
         uses: actions/setup-dotnet@v3.2.0
         with:
-          dotnet-version: 8.0.x
+          dotnet-version: |
+            8.0.x
+            7.0.x
```

Comment:
Why is 7 needed? I don't see a reference to is in the changes.

Reply:
To give RavenDB exactly what it wants. Even though the PR to step up to .NET 8 SDK worked in CI, in 90% of CI runs after that, Raven was failing to start in less than 30 seconds on one of the test projects. :shrug: